boto3/docs/service.py

```diff
@@ -84,10 +80,39 @@ def client_api(self, section):
             self._client, self._root_docs_path, examples
         ).document_client(section)
 
+    def resource_section(self, section):
+        section.style.h2('Resources')
+        section.style.new_line()
+        section.write(
+            f'The available resources are:'
+        )
+        section.style.new_line()
+        section.style.toctree()
+        self._document_service_resource(
+            section
+        )
+        self._document_resources(section)
+
     def _document_service_resource(self, section):
+        # Create a new DocumentStructure for each Service Resource and add contents.
+        service_resource_doc = DocumentStructure('ServiceResource', target='html')
```

Comment:
It looks like we've converted `ServiceResource` to `service-resource` everywhere else. Is there a reason it's still CamelCase here?

Reply:
For the reasons in my [previous response](https://github.com/boto/boto3/pull/3597#discussion_r1110317266), this had to be 'serviceresource' or 'service-resource'. I thought the latter looked better and also correlated with how the Service Resource section fragment looks `#service-resource`. Either one can be supported.

Reply:
Yep, I think that's fine with the explanation above. Let's make it all consistent with what you have and we can call it good.